mappings/net/minecraft/world/World.mapping

```diff
@@ -36,41 +159,156 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 		ARG 1 entityBoundingBox
 		ARG 2 fluidMaterial
 		ARG 3 entity
-	METHOD method_172 (Lnet/minecraft/class_54;IIII)V
+	METHOD method_171 canInteract (Lnet/minecraft/class_54;III)Z
+		ARG 1 player
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
+	METHOD method_172 extinguishFire (Lnet/minecraft/class_54;IIII)V
+		ARG 1 player
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 direction
+	METHOD method_173 worldEvent (Lnet/minecraft/class_54;IIIII)V
+		ARG 1 player
+		ARG 2 event
+		ARG 3 x
+		ARG 4 y
+		ARG 5 z
+		ARG 6 data
+	METHOD method_174 countEntities (Ljava/lang/Class;)I
+		ARG 1 entityClass
+	METHOD method_175 collectEntitiesByClass (Ljava/lang/Class;Lnet/minecraft/class_25;)Ljava/util/List;
+		ARG 1 entityClass
+		ARG 2 box
 	METHOD method_176 getOrCreateState (Ljava/lang/Class;Ljava/lang/String;)Lnet/minecraft/class_58;
 		ARG 1 stateClass
 		ARG 2 id
+	METHOD method_177 getPlayer (Ljava/lang/String;)Lnet/minecraft/class_54;
+		ARG 1 name
 	METHOD method_178 addParticle (Ljava/lang/String;DDDDDD)V
+		ARG 1 particle
+	METHOD method_179 playMusic (Ljava/lang/String;III)V
```

Comment:
This seems incorrect. This would make me think that it would play background music from `/music` or `/newmusic` directories. When it actually plays from `/streaming` because this is the function for playing records I believe. The names of both functions should be `playStreaming` or `playRecord` or something

Reply:
I'll make it playStreaming